.github/workflows/auto-security-license.yml

```
@@ -0,0 +1,125 @@
+              "LGPL-2.1",
+              "LGPL-3.0",
+              "MPL-2.0",
+              "CDDL-1.0",
+              "EPL-1.0",
+              "EPL-2.0"
+            ]
+          }
+          EOF
+
+          echo "📋 Checking license compliance..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)